src/extension.ts

```diff
@@ -56,9 +56,20 @@ export function activate(context: vscode.ExtensionContext) {
 		await node.createChild();
 	});
 	actionHandler.registerCommand('cosmosDB.attachDatabaseAccount', async () => {
-		const attachedNode: IAzureParentNode<AttachedAccountsTreeItem> = await getAttachedNode(tree);
-		await attachedNode.treeItem.attachNewAccount();
-		await tree.refresh(attachedNode);
+		const rootNodes = await tree.getChildren();
+		const attachedAccountsNode = <IAzureParentNode<AttachedAccountsTreeItem>>rootNodes.find((node) => node.treeItem instanceof AttachedAccountsTreeItem);
```

Comment:
Use `getAttachedNode` (Although you can change the name to `getAttachedAccountsNode` if you want)

In addition, the node should always exist, so you don't need to check `if (attachedAccountsNode)`